maintenance/oncall.html

```diff
@@ -2,6 +2,25 @@
 <!-- To "deploy": gsutil cp -Z oncall.html gs://test-infra-oncall/ -->
 <html lang="en">
 <script>
+  var rotationsMetadata = {
+    "google-build-admin": {
+      "prettyName": "Google Build Admin",
+      "slack": "@google-build-admin",
+      "timezone": '<a href="https://time.is/PT">PT</a>',
+      "description": 'Debian / RPM package build & publishing for releases',
+    },
+    "testinfra": {
+      "prettyName": "Test Infra",
+      "slack": "@test-infra-oncall",
+      "timezone": '<a href="https://time.is/PT">PT</a>',
+      "description": "Prow, TestGrid, and other Google hosted infrastructure"
+    },
+    "scalability": {
+      "prettyName": "Scalability",
+      "timezone": '<a href="https://time.is/CET">CET</a>',
+      // TODO: description
```

Comment:
@wojtek-t @tosi3k we could use a short description of the scalability rotation here
for slack see: https://github.com/kubernetes/test-infra/pull/18458#discussion_r459772745